File: cowork/Domain/Cowork/MealBooking.cs
namespace coworkdomain.Cowork {

    public class MealBooking {

        public MealBooking() { }

        public MealBooking(long id, long mealId, long userId, string note) {
            Id = id;
            MealId = mealId;
            UserId = userId;
            Note = note;
        }

        public long Id { get; set; }
        public long MealId { get; set; }
        public long UserId { get; set; }
        public string Note { get; set; }
        
        public Meal Meal { get; set; }
        public User User { get; set; }

    }

}

File: cowork/ClientApp/src/app/models/MealBooking.ts
import {Meal} from './Meal';
import {User} from './User';

export class MealBooking {
    public Id: number;
    public MealId: number;
    public UserId: number;
    public Note: string;
    public Meal: Meal;
    public User: User;

    constructor(id = -1, mealId = null, userId = null, note = "") {
        this.Id = id;
        this.MealId = mealId;
        this.UserId = userId;
        this.Note = note;
    }
}
